# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/lib/prologue.js
# Commit: 453aa2faad97
# Full Hash: 453aa2faad97ce68b33bef75cd232e37d759556e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-29 09:50:10
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r d33708e4f8ec -r 453aa2faad97 js/src/jit-test/lib/prologue.js
--- a/js/src/jit-test/lib/prologue.js	Fri Oct 28 15:17:43 2022 +0000
+++ b/js/src/jit-test/lib/prologue.js	Fri Oct 28 15:17:43 2022 +0000
@@ -16,7 +16,8 @@
                     "selectforgc",
                     "verifyprebarriers",
                     "verifypostbarriers",
-                    "gcPreserveCode"]) {
+                    "gcPreserveCode",
+                    "setMarkStackLimit"]) {
     const present = name in this;
     if (!present) {
         this[name] = function() {};